docs/_sections/forms.html

```diff
@@ -89,30 +89,30 @@ <h3 id="requiredFields">Required fields</h3>
 
     <h3 id="fieldErrors">Error state on fields</h3>
 
-    <p>Any field where user input could be deemed or returned erroneous should utilize the class <code>nypl-field-error</code>. It is up to the form designer to determine what fields should have an error state. It is up to the developer to implement the necessary front and back end validations as well as the expected input formatting that would necessitate a state change for a given field. Additionally, explanatory text must be supplied by the form designer decribing the error and how to correct it. </p>
-
+    <p>Any field where user input could be returned erroneous should utilize the class <code>nypl-field-error</code>. The developer must append the <code>span</code> with the class: <code><span>error-desc</span></code> upon validation. It is up to the product designer to determine what fields should have an error state and provide correct explanatory text. It is the responsibility of the developer to implement the necessary front and back end validations, via Javascript or some other means.</p>
+    <p class="nypl-a11y-note"><strong>Accessibility note:</strong> The correct <code>aria</code> attributes must be added to the <code>input</code> field for proper accessibility coverage. The example below describes both <code>aria-requried</code> and <code>aria-invalid</code>. <code>aria-requried</code> will be a static state for the form, whereas <code>aria-invalid</code> will initially be set to <code>false</code>, upon a an invalid submission, the attribute should return <code>true</code> as it's value. See the following resources:
+      <ul>
+        <li>
+          <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/forms/Basic_form_hints#Required_and_invalid_fields" target="_blank">Basic Form Hints: Required and invalid fields</a>
+        </li>
+      </ul>
+    </p>
     <div class="nypl-toolkit-example">
       <span class="nypl-example-text">Example</span>
       <div class="nypl-text-field nypl-field-error">
-        <label for="keyword-text">Title
+        <label for="keyword-text">Email
```

Comment:
fix the `for`s so that they point to the right fields (applies to required field also)